docs/features/techdocs/concepts.md

```diff
@@ -8,13 +8,41 @@ Spotify's docs-like-code solution in Backstage
 This page describes concepts that are introduced with Spotify's docs-like-code
 solution in Backstage.
 
-### TechDocs Core Plugin
+### TechDocs Preparer
 
-The TechDocs Core Plugin is an [MkDocs](https://www.mkdocs.org/) plugin created
-as a wrapper around multiple MkDocs plugins and Python Markdown extensions to
-standardize the configuration of MkDocs used for TechDocs.
+Prepare is the first step of generating documentation for an entity. It fetches
```

Comment:
Maybe use the same form of the word in all of these: Preparing, Generating, Publishing.